webapp/loggedin/appHeader/components/userPopupMenu.scss

```diff
@@ -26,6 +26,7 @@
   img {
     border-radius: 100%;
     width: 100%;
+    height: auto;
```

Comment:
I remember I removed auto because it has a strange effect in Chrome. Can you please check? I'm thinking maybe it's better to give height and width in px? What do you think?